src/_data/home.json

```diff
@@ -1,44 +1,44 @@
+    "title": "Custom solutions for your company",
     "content": "FlowFuse offers the customization essential for your specific needs. Leveraging Node-RED and its extensive library, we enable the creation of manufacturing applications that align perfectly with your operational requirements.",
     "image": "bespoke_red",
     "imagealt": "icon of code"
   },
   {
-    "title": "<span class=\"text-red-600\">Centralized<\/span> management and automated deployment",
+    "title": "Centralized management and automated deployment",
     "content": "FlowFuse streamlines the management of your development environments, offering a unified platform for deployment and configuration. With our automated processes, creating and deploying configurations across multiple devices is efficient and hassle-free.",
     "image": "scalable_pictogram_red",
     "imagealt": "scalability"
   },
   {
-    "title": "<span class=\"text-red-600\">Reliable delivery<\/span> every time",
+    "title": "Reliable delivery every time",
     "content": "FlowFuse revolutionizes the software development process, streamlining the lifecycle for your applications. Our DevOps pipelines facilitate development, testing, and production, ensuring consistent and reliable application delivery.",
-    "image": "pipeline_red",
+    "image": "small-pipeline_red",
     "imagealt": "a pipeline of Node-RED instances"
   },
   {
-    "title": "<span class=\"text-red-600\">Secure and compliant<\/span> development",
+    "title": "Secure and compliant development",
     "content": "Digitally transform your operations while maintaining the security you rely upon. Single-Sign-On (SSO), audit logs for monitoring, and change control are a few ways we ensure secure building in a collaborative environment.",
     "image": "security_pictogram_red",
     "imagealt": "a lock"
   },
   {
-    "title": "<span class=\"text-red-600\">Low-code<\/span> development",
-    "content": "Our low-code platform enables your team to rapidly develop tailored applications in-house for faster, optimized results.",
-    "image": "citizen_dev_red",
-    "imagealt": "a team of developers"
-  },
-  {
-    "title": "<span class=\"text-red-600\">Expert Support<\/span> for your applications",
+    "title": "Expert Support for your applications",
     "content": "FlowFuse is your trusted partner for professional support, ensuring smooth and secure operation of your critical applications. Our team of experts is dedicated to tackling any challenge and maintaining the highest level of performance. We keep your systems current with the latest updates, safeguarding the continuity and efficiency of your operations.",
     "image": "support_pictogram_red",
     "imagealt": "support"
+  },
+  {
+    "title": "Low-code development",
+    "content": "Our low-code platform enables your team to rapidly develop tailored applications in-house for faster, optimized results.",
```

Comment:
@Hasmin-AC Can you create a PR to add more words to this? The homepage looks a bit odd with just this one liner.

Reply:
Since this is a odd point from the 2 columns, can we consider this point as its own section - "Low-code Development" becomes a main title and we add a screenshots or animated diagram of platform?